python/plugins/db_manager/db_plugins/oracle/plugin.py

```diff
@@ -263,6 +261,9 @@ def registerDatabaseActions(self, mainWindow):
         mainWindow.registerAction(action, QApplication.translate(
             "DBManagerPlugin", "&Table"), self.emptyTableActionSlot)
 
+    def searchClass(self):
```

Comment:
Remove all those searchClass methods. All logic should stay encapsulated here.

Reply:
searchClass must be replaced by supportComment() rigth ?

Reply:
yes